src/App.tsx

```diff
@@ -15,48 +17,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  noSort = '',
+  leng = 'leng',
+  alph = 'alph',
+}
+
+interface StateInterface {
+  sortType: SortType;
+  isReverse: boolean;
+}
+
+const updateGoods = (
+  goods: string[], { sortType, isReverse }: StateInterface,
```

Comment:
Why do you pass the second and third arguments as one object? 
What is the point in this?

Reply:
To ensure that the order in which the sort parameters are passed is unimportant.

Reply:
@Wesses There aren't that many arguments here to worry about a possible error in the order of arguments.
In addition, you still retain the importance of order (the array must always be first), so if you want to be sure that the order of the arguments is not important, you should pass all the arguments as one object, not just a part.